tests/cmd/upgrade/preserve_precision_minor.toml

```diff
@@ -0,0 +1,13 @@
+bin.name = "cargo-upgrade"
+args = ["upgrade", "--preserve-precision"]
+status = "success"
+stdout = """
+cargo-list-test-fixture:
+"""
+stderr = """
+    Upgrading docopt v0.8 -> v99999.0
```

Comment:
I'm curious, how was your experience with `trycmd`?

Reply:
Admittedly, I don't understand what all of the options do. But it was pretty easy for me to copy one of the existing tests and modify it for my purposes. It only took a few minutes.